doc/rados/operations/health-checks.rst

```diff
@@ -141,6 +141,14 @@ To adjust the warning threshold, run the following command:
 
    ceph config set global mon_data_size_warn <size>
 
+MON_NETSPLIT
+____________
+
+A network partition has occurred between the monitors. This health check is
+raised when one or more monitors detect that at least one pair of monitors
+has lost connectivity, based on their individual ``connection score``,
+which are consistently shared and updated. This warning only appears when
+we have ``at least 3 monitors`` using ``connectivity election strategy``.
```

Comment:
s/we/the cluster is provisioned with at least three Ceph Monitors and are using the/

I think no backticks